DependencyInjection/Configuration.php

```diff
@@ -37,7 +37,7 @@ public function getConfigTreeBuilder()
                     ->addDefaultsIfNotSet()
                     ->canBeUnset()
                     ->children()
-                        ->scalarNode('template')->defaultValue('knp_menu.html.twig')->end()
+                        ->scalarNode('template')->defaultValue('KnpMenuBundle::menu.html.twig')->end()
```

Comment:
ah sorry, found it. you create a new template extending that one.

why do you extend it - could this also go into the base KnpMenu? the template does not look symfony specific to me. or am i missing something?